packages/run-protocol/src/econ-behaviors.js

```diff
@@ -251,6 +253,7 @@ export const startVaultFactory = async (
       recordingPeriod: SECONDS_PER_DAY,
     },
   } = {},
+  minInitialDebt = 5_000_000n,
```

Comment:
I tried to do that, but centralBrandP is coming in as a separate parameter, so AFAICT it can't be used to construct a default value for this parameter.